src/services/gallery.ts

```diff
@@ -63,9 +68,16 @@ export const getSearchedValue = (filters: Filter[]) =>
     .map((f: Filter) => f.value)
     .toString();
 
+export const getCapabilityFiltered = (filters: Filter[]) =>
+  filters
+    .filter((f: Filter) => f.name === 'capability')
+    .map((f: Filter) => f.value)
+    .toString();
```

Comment:
Le `toString()` est-il nécessaire ?

Reply:
Alors autant pour la barre de recherche, non (étant donné qu'on ne fait qu'afficher le retour dans la barre de recherche)
Dans le cas présent, oui et non :D

le `map` me retourne un `string[]`, qui ne peut pas être comparé avec une valeur au niveau du component 

Actuellement, je fais cela 
`<Radio key={value} checked={getCapabilityFiltered(stateFilters) === value} name="capability" value={value} label={value} capitalize={true} />`

Pour passer en checked la capability sélectionnée.
Donc, je vois 2 autres moyens de le faire (mais ca change pas grand chose in fine)
- remplacer le `toString()` par un `.reduce((prev, curr) => (prev + '' + curr));`   ==> pas sûr qu'on gagne en lisibilité
- décaler le `toString()` dans le component 
```
<Radio key={value} checked={getCapabilityFiltered(stateFilters) === value} name="capability" value={value} label={value} capitalize={true} />
```
deviendrait alors 
```
<Radio key={value} checked={getCapabilityFiltered(stateFilters).toString() === value} name="capability" value={value} label={value} capitalize={true} />
```